prediction/requirements/local.txt

```diff
@@ -1,4 +1,4 @@
 -r base.txt
 flake8==3.3.0
 pytest==3.1.3
-pylidc==0.1.8
+pylidc==0.1.9
```

Comment:
This fixed a bug in `to_volume`, which we'll need later.

Reply:
Hm, I wonder if there's a better way of storing these kind of comments. Can we do 

```
pylidc==0.1.9 # comment here
```

or at *least* put it in the commit. Putting in the **PR review** is really a poor place for it IMHO.